cli/index.js

```diff
@@ -11,6 +11,7 @@ const mediaMinMax = require('postcss-media-minmax');
 const customMedia = require('postcss-custom-media');
 const imports = require('postcss-easy-import');
 
+const src = require('./src');
```

Comment:
@keenanpayne There isn't a src or src/index.js file in this path, was this line intended for something or you just forgot to remove it?

Reply:
This is intended to watch the source .pcss files to determine if any of them have changed so that we can know whether to trigger a re-compile.

Reply:
Oh I see what you mean, you use that value in line 61 as `watch(src)`. But I think for the function `watch()` that you created we only need a path, which we can obtain with  `path.dirname(command.input)`. By doing `require()` it looks for a JavaScript file and there isn't one. What do you think?

Reply:
I'll finish something I'm doing and will upload changes with this one, so you can check it out

Reply:
It's up

Reply:
Great, thank you for making the improvement! That makes more sense to me